documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -14,8 +14,10 @@ Install with `npm i -D @sveltejs/adapter-static`, then add the adapter to your `
 // @errors: 2307
 /// file: svelte.config.js
 import adapter from '@sveltejs/adapter-static';
+import preprocess from 'svelte-preprocess';
```

Comment:
We probably should use `vitePreprocess` here to be in line with the starter template